crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -1206,6 +1207,87 @@ mod test {
         conductor_handle.shutdown().await.unwrap().unwrap();
     }
 
+    #[tokio::test(flavor = "multi_thread")]
+    async fn dump_conductor_state() {
+        holochain_trace::test_run();
+        let uuid = Uuid::new_v4();
+        let dna = fake_dna_zomes(
+            &uuid.to_string(),
+            vec![("zomey".into(), TestWasm::Foo.into())],
+        );
+        let agent_pubkey = fake_agent_pubkey_1();
+
+        let _ = RealRibosomeFixturator::new(crate::fixt::curve::Zomes(vec![TestWasm::Foo]))
+            .next()
+            .unwrap();
+        let (_tmpdir, _, conductor_handle, _agent_key) = setup_app_in_new_conductor(
+            "test app".to_string(),
+            Some(agent_pubkey.clone()),
+            vec![(dna.clone(), None)],
+        )
+        .await;
+        let dna_hash = dna.dna_hash();
+
+        conductor_handle
+            .clone()
+            .add_app_interface(
+                either::Either::Left(12345),
+                AllowedOrigins::Any,
+                Some("test app".into()),
+            )
+            .await
+            .unwrap();
+
+        // Construct expected response
+        #[derive(Serialize, Debug)]
+        pub struct ConductorSerialized {
+            running_cells: Vec<(DnaHashB64, AgentPubKeyB64)>,
+            shutting_down: bool,
+            admin_websocket_ports: Vec<u16>,
+            app_interfaces: Vec<AppInterfaceId>,
+        }
+
+        #[derive(Serialize, Debug)]
+        struct ConductorDump {
+            conductor: ConductorSerialized,
+            state: ConductorState,
+        }
+
+        let dpki_cell_id = conductor_handle
```

Comment:
This test doesn't look like it's behind a feature flag for DPKI, is the DPKI cell still running? Or is this to make the test work when the DPKI feature is enabled?

Reply:
Yeah this makes the test work with dpki enabled and without. I could put this bit of code behind the feature flag if desired.

Reply:
All good, just wanted to check that DPKI isn't quietly running when it's disabled :D